.github/workflows/python-package.yml

```diff
@@ -129,7 +129,7 @@ jobs:
         sudo apt-get install lcov
     - name: Install python build dependencies
       run: |
-        python -m pip install --upgrade pip setuptools wheel flake8>=3.5 check-manifest
+        python -m pip install --upgrade pip setuptools wheel "flake8>=3.5" check-manifest
```

Comment:
I wonder if we might need to freeze these as well.

Reply:
I think these are fine. If we were to bake them, it would not necessarily be here, but in the pyproject.toml. But one problem with that is that dependabot doesn't know yet how to automatically update a pyproject.toml (though GH are implementing this as I write this and I suspect it will become available in a couple of weeks).

Reply:
Ok cool. In the future!